book/src/SUMMARY.md

```diff
@@ -13,6 +13,20 @@
     - [Git hooks](development/git-hooks.md)
     - [Styling](development/styling.md)
     - [Making a release](development/releasing.md)
+- [Components](components/README.md)
+    - [CatalogueComponent](components/catalogue.md)
+    - [ChartComponent](components/chart.md)
+    - [InfoButtonComponent](components/infobutton.md)
+    - [modifiedsearchComponent](components/modifiedsearch.md)
+    - [NegotiateButtonComponent](components/negotiatebutton.md)
+    - [QueryExplainButtonComponent](components/queryexplainbutton.md)
+    - [QuerySpinnerComponent](components/queryspinner.md)
+    - [ResultsummeraryComponent](components/resultsummary.md)
+    - [ResultTableComponent](components/resulttable.md)
+    - [SearchbarComponent](components/searchbar.md)
+    - [SearchbarComponentMultiple](components/searchbarmulti.md)
+    - [SearchButtonComponent](components/searchbutton.md)
+    - [StoreDeleteButtonComponent](components/storedeletebutton.md)
```

Comment:
Change from camel case to human readable text, e.g. "ResultTableComponent" to "Result Table"